From: Miaohe Lin <linmiaohe@huawei.com>
Date: Tue, 7 Jun 2022 22:36:21 +0800
Subject: mm/memremap: fix wrong function name above memremap_pages()
Git-commit: 23689037e0986724e3bdc41bb2ee6fa1b497b8f9
Patch-mainline: v6.0-rc1
References: jsc#PED-1294

Fix the wrong function name dev_memremap_pages above memremap_pages() to
avoid confusion.  Minor readability improvement.

Link: https://lkml.kernel.org/r/20220607143621.58989-1-linmiaohe@huawei.com
Signed-off-by: Miaohe Lin <linmiaohe@huawei.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Vlastimil Babka <vbabka@suse.cz>
---
 mm/memremap.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/mm/memremap.c
+++ b/mm/memremap.c
@@ -278,8 +278,8 @@ err_pfn_remap:
 
 
 /*
- * Not device managed version of dev_memremap_pages, undone by
- * memunmap_pages().  Please use dev_memremap_pages if you have a struct
+ * Not device managed version of devm_memremap_pages, undone by
+ * memunmap_pages().  Please use devm_memremap_pages if you have a struct
  * device available.
  */
 void *memremap_pages(struct dev_pagemap *pgmap, int nid)
